packages/core/directives/polyline.ts

```diff
@@ -78,6 +78,11 @@ export class AgmPolyline implements OnDestroy, OnChanges, AfterContentInit {
    */
   @Input() strokeWeight: number;
 
+  /**
+   * The icons to be rendered along the polyline.
+   */
+  @Input() icons: Array<IconSequence>;
```

Comment:
Is it something that can change at runtime? Than we need to add it to the `_polylineOptionsAttributes`

Reply:
Oh yes. Thanks for reviewing it. I have made the code changes and committed it.